_overviews/scala3-scaladoc/blog.md

```diff
@@ -33,4 +33,29 @@ Scaladoc loads blog if the `_blog` directory exists.
 All the blogpost filenames should start with date in numeric format matching `YYYY-MM-DD`.
 Example name is `2015-10-23-dotty-compiler-bootstraps.md`.
 
+## Structure
 
+When creating a blog page, you will have a first part to fill in with different fields. See the following example:
+
+```
+---
+layout: <A reference to the layout page for the blog page>
+author: <Name of the author of the page>
+title: <Title of the page>
+subTitle: <Subtitle of the page>
```

Comment:
Any idea what this will be used for?

Reply:
It just puts a subtitle like this, do you think it's a good idea if I put an example picture underneath like this one?
<img width="772" alt="Screenshot 2023-03-02 at 13 41 57" src="https://user-images.githubusercontent.com/44496264/222432071-dd2b37b4-ed67-48ae-988e-41709f559e7f.png">

Reply:
ah no this one is fine.